.gitignore

```diff
@@ -68,3 +68,4 @@ util/Win_Check_Output.txt
 secrets.tar
 id_rsa_*
 /.vs
+*.gch
```

Comment:
@jackhumbert or @skullydazed is this addition okay?

Reply:
I can remove this change if necessary.

Reply:
Yes, if you could, please.